jhove-core/src/main/java/edu/harvard/hul/ois/jhove/Property.java

```diff
@@ -77,7 +77,12 @@ private void init (String name, PropertyType type, PropertyArity arity,
            types here may help avoid difficult chasing down
            of the bugs such mismatches cause. */
 	if (value == null) {
-	    throw new NullPointerException (CoreMessageConstants.EXC_PROP_VAL_NULL);
+		if (type == PropertyType.STRING) {
+			// a String value can be empty
+			value = "";
+		} else {
```

Comment:
The else is unnecessary if it's not a string then we always want the exception which terminates.